### ISSUE TITLE
Phenotyper line on graph stops at 5 dimensions

### ISSUE THREAD

**Issue by bjoyce3** [rseng/rse-phenotype#9]
I really like this tool, but I fear I've broken it. 

You can create more than 5 dimensions, but the lines on the graph won't make more than 5 points. I believe from a small amount of testing I can confirm that the last dimensions in the list are the ones that don't get dots/lines created. In the attached screen shot the last two items in the list (Grants/Proposals and Firefighting) are the ones that also don't get dots to give a percentage. 

Two potential fixes out of infinite potential fixes:
- don't let anyone put in more than 5 dimensions and thus the dots won't matter
- let people put in more than 5 dimensions but figure out why the dots/lines aren't generated (and generate them I suppose).

![Screen Shot 2020-07-09 at 20 30 44 (2)](https://user-images.githubusercontent.com/11023317/87113674-a53b6b00-c223-11ea-948c-1e9afc7dad52.png)

**Comment by vsoch** [comment]
hey @bjoyce3! Thanks for reporting the issue - it's definitely a bug! I've transferred the issue to the repository here since this is where the code is served from. We definitely don't want to limit the number of dimensions that the user can define, so best to just fix the bug. I've done this in https://github.com/rseng/rse-phenotype/pull/10 if you would care to test, you should be able to clone the branch, and then start a local webserver. Eg.,

```bash
git clone -b fix/dimension-bug https://github.com/rseng/rse-phenotype
cd rse-phenotype

# open browser to port 9999
python -m http.server 9999 
```
Let me know if that fixes it for you and I'll merge the PR.

**Comment by vsoch** [comment]
Fixed with #10 (I tested it).
